aws/velero-bucket/outputs.tf

```diff
@@ -1,17 +1,17 @@
 output "bucket_name" {
-  value = element(concat(aws_s3_bucket.this.*.id, list("")), 0)
+  value = element(concat(aws_s3_bucket.this.*.id, tolist([""])), 0)
```

Comment:
I'd prefer `try(element(aws_s3_bucket.this.*.id, 0), "")`. I feel that this makes the intent of the expression much clearer. Same for the other outputs.

(Either way, `tolist()` is redundant here.)

Reply:
I think `try(aws_s3_bucket.this.*.id[0], "")` also works.

Reply:
let me try

Reply:
it worked, just updated.